internal/crypto/utils.go

```diff
@@ -0,0 +1,9 @@
+package crypto
+
+func must[T any](a T, err error) T {
+	if err != nil {
+		panic(err)
+	}
+
+	return a
+}
```

Comment:
might be a good idea to move this into `utilities` package instead and import it in `crypto`?